prep-javascript-variables/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Brittany Boughter';
+    console.log(fullName);
+    var numberOfPets = '2';
```

Comment:
We want to leave the quotes off of our number values so that the javascript engine doesn't interpret them as strings.

Reply:
![prep-javascript-variables](https://user-images.githubusercontent.com/113574704/192586695-fb0d855d-4844-447e-ad0c-c4156b7cb6b8.PNG)